self-tests.yml

```diff
@@ -644,6 +644,29 @@ buildvariants:
       - name: ".test"
       - name: test-db-auth
 
+  - name: ubuntu2004-container
+    display_name: Ubuntu 20.04 (Container)
+    run_on:
+      - evg-container
+    expansions:
+      goos: linux
+      goarch: amd64
+      IS_DOCKER: true
+      GOROOT: /usr/local/go
+      mongodb_url: https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu2004-5.0.14.tgz
+      decompress: tar zxvf
+    tasks:
+      - name: "dist-and-upload-clis"
+      - name: "dist-staging"
```

Comment:
Do we need this? Seems like it's not that useful since we already run `dist-unsigned`. `dist-staging` is only used because some devs preferred to run patches to create staging dists.

Reply:
Was trying to replicate our 20.04 variant but yeah that sounds reasonable. Removed